core-server/src/main/resources/org/glassfish/jersey/server/internal/localization.properties

```diff
@@ -71,6 +71,7 @@ error.resources.cannot.merge=Resources do not have the same path and cannot be m
 error.request.abort.in.response.phase=The request cannot be aborted as it is already in the response processing phase.
 error.request.set.entity.stream.in.response.phase=The entity stream cannot be set in the request as it is already in the response processing phase.
 error.request.set.security.context.in.response.phase=The security context cannot be set in the request as it is already in the response processing phase.
+error.response.already.commited=Response was already committed. This response is being reused more than 1 time.
```

Comment:
The response entity was already committed. The Response entity is being reused more than once.